CODE_OF_CONDUCT.md

```diff
@@ -8,6 +8,8 @@ Examples of unacceptable behavior by participants include the use of sexual lang
 
 Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct. Project maintainers who do not follow the Code of Conduct may be removed from the project team.
 
+This code of conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.
```

Comment:
"when" or "where"? When reads klunky to my ear, but both are communicating similar/same concepts.